File: spec/python/test_repeat_n_strz_double.py
import unittest

from repeat_n_strz_double import RepeatNStrzDouble

class TestRepeatNStrzDouble(unittest.TestCase):
    def test_repeat_n_strz_double(self):
        r = RepeatNStrzDouble.from_file("src/repeat_n_strz.bin")

        self.assertEqual(r.qty, 2)
        self.assertEqual(r.lines1, ["foo"])
        self.assertEqual(r.lines2, ["bar"])

File: spec/javascript/test_repeat_n_strz_double.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('RepeatNStrzDouble', 'src/repeat_n_strz.bin', function(r) {
  assert.equal(r.qty, 2);
  assert.deepEqual(r.lines1, ['foo']);
  assert.deepEqual(r.lines2, ['bar']);
});
